charlib/characterizer/port.py

```
@@ -106,7 +106,26 @@ def __init__(self, noninverting_port_name: str, inverting_port_name: str, direct
+            return item in [self.noninverting_port_name, self.inverting_port_name]
+        elif isinstance(item, Pin):
+            return item in list(self.as_pins())
+        return False
+
+    def complement(self, port_name):
+        """Given the name of one port in the pair, return the other port.
+
+        Returns None if port_name is not in the DifferentialPair."""
+        if port_name == self.noninverting_port_name:
+            return self.inverting_port_name
+        elif port_name == self.inverting_port_name:
```

Ruff (0.15.17)

[warning] Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)